doc/source/whatsnew/v1.0.0.rst

```diff
@@ -249,6 +249,7 @@ Bug fixes
 ~~~~~~~~~
 
 - Bug in :meth:`DataFrame.to_html` when using ``formatters=<list>`` and ``max_cols`` together. (:issue:`25955`)
+- Bug in :func:`pandas.core.generic.NDFrame.interpolate` where specifying axis by name references variable before it is assigned (:issue:`29132`)
```

Comment:
These are user facing, so we shouldn't reference NDFrame. Instead, refer to just :meth:\`DataFrame.interpolate\`, even if it affects Series as well.

Reply:
I see!